JavaScript/js/gameDefinition.js

```diff
@@ -1,13 +1,23 @@
-// Jacob's working on this...
-
+/**
+ * Main game object that controls game flow.
+ */
 class Game {
+
+  /**
+   * @param setupData An object specifcally formatted to provide info to the game.
+   * See config.js for example structure.
+   */
   constructor(setupData) {
-    //setup the DOM with jquery
+    // Link the DOM command input area using jQuery
     const $userInput = $("#command")
-    const $outputBox = $("#message")
 
-    //create a messenger with access to the dom output box
-    const messenger = new Messenger($outputBox)
+    // Link the DOM output areas using jQuery
+    const $messageArea = $("#message")
+    const messenger = new Messenger($messageArea)
+    const $nameArea = $("#name")
+    const roomNamer = new Descriptor($nameArea)
+    const $descriptionArea = $("#description")
```

Comment:
What is the "description" ID in context of the DOM? What type of descriptions will be viewed there?

Isn't the message area what shows descriptions when a user enters a room? Usually the way text based adventures behave, what do you think?